src/Octoshift/Mannequins.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using OctoshiftCLI.Models;
+
+namespace Octoshift
+{
+    public class Mannequins
```

Comment:
should this be in the Models folder?

Reply:
Yeah, I struggled a bit on that one....

Moved to models

Reply:
I swear GitHub ate my comment 😢  about this. anyways, clearly this class is not a model and is a service since it bundled bunch of helper methods.